projects/dav1d/build.sh

```diff
@@ -23,8 +23,13 @@ rm -rf ${build}
 mkdir -p ${build}
 
 # build library
+if [ "$ARCHITECTURE" = "x86_64" ]; then
+	use_asm=true
+else
+	use_asm=false
+fi
```

Comment:
Is this necessary? The main difference between x86_64 and i386 would be in the asm. If it is necessary to build i386 I would prefer to fix it instead. Is there information available on how oss-fuzz builds for i386? Does it just adds `-m32` to CFLAGS/LFLAGS? I'll check tonight (central european summer time).

Reply:
So it only adds `-m32` to CFLAGS. this doesn't work for meson. I'll open a merge request with meson "cross" compiling support  to supecede this MR.